tests/test-sources/plugins/by-name/wrapping/default.nix

```diff
@@ -0,0 +1,11 @@
+{ pkgs, ... }:
+{
+  empty = {
+    plugins.wrapping.enable = true;
+  };
+
+  defaults.plugins.which-key = {
+    enable = true;
+    notify_on_switch = true;
```

Comment:
```suggestion
    settings.notify_on_switch = true;
```

or

```suggestion

    settings = {
      notify_on_switch = true;
    };
```

Reply:
This is still not resolved:
```
error: The option `plugins.which-key.notify_on_switch' does not exist. Definition values:
- In `/nix/store/XXX-test-sources/plugins/by-name/wrapping/default.nix': true
```

Reply:
Thanks, just did that and had fun running the specified tests :)